### DISCUSSION TITLE
Is it compare with Windows 11 ARM?

### DISCUSSION THREAD

**Discussion by codezi** [valinet/ExplorerPatcher#632] [General]
Hello,
I am using Windows 11 ARM Parallels on my Macbook. And I install the Explorer patcher and nothing changes.
Am I missing something?
Thanks.

**Comment by valinet** [comment]
ExplorerPatcher is compiled for the amd64 architecture only. No version for ARM64 is available at the moment, as I don’t have such a machine to test on and no one else has yet to come with a working port. Emulation doesn’t cut it for this kind of app, a native version is necessary. More details on the bottom of [this page](https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems). Thanks.

**Reply by codezi** [reply]
Thank you so much for your reply.
